test/Microsoft.AspNetCore.OData.Versioning.ApiExplorer.Tests/AspNet.OData/Contact.cs

```diff
@@ -1,18 +1,25 @@
-﻿namespace Microsoft.AspNet.OData
+﻿using System.Runtime.Serialization;
+
+namespace Microsoft.AspNet.OData
 {
     using System.Collections.Generic;
-
+    [DataContract]
     public class Contact
     {
+        [DataMember]
         public int ContactId { get; set; }
-
+        [DataMember(Name = "first_name")]
```

Comment:
There should be line separating these two properties.

Reply:
Fixed